master/buildbot/steps/artifact.py

```diff
@@ -183,9 +187,15 @@ def start(self):
 
         if self.artifactBuildrequest:
             self.step_status.setText(["Artifact has been already generated."])
-            self.artifactPath = "%s_%s_%s" % (self.build.builder.config.builddir,
-                                              self.artifactBuildrequest['brid'],
-                                              FormatDatetime(self.artifactBuildrequest['submitted_at']))
+
+            if self.artifactBuildrequest["submitted_at"] > ARTIFACT_LOCATION_CHANGE_DATE:
+                self.artifactPath = "%s/%s_%s" % (self.build.builder.config.builddir,
```

Comment:
@campos-ddc why don't we suppor this format right away, is there a particular reason why we should wait for the specific date?